plugins/toolbox-search/src/toolbox_search.ts

```diff
@@ -131,7 +131,7 @@ export class ToolboxSearchCategory extends Blockly.ToolboxCategory {
    */
   private initBlockSearcher() {
     const availableBlocks = new Set<string>();
-    this.workspace_.options.languageTree.contents.map((item) =>
+    this.workspace_.options.languageTree?.contents?.map((item) =>
```

Comment:
This is unrelated, but can you switch this to `forEach` instead of `map`? We shouldn't be using `map` if we're not using the return value.

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#:~:text=Since%20map%20builds%20a%20new%20array%2C%20calling%20it%20without%20using%20the%20returned%20array%20is%20an%20anti%2Dpattern%3B%20use%20forEach%20or%20for...of%20instead.

Reply:
Went through the reference your provided and updated the same.